examples/simple/main.go

```diff
@@ -66,6 +66,11 @@ func main() {
 	}
 
 	for _, tracker := range s.GetTracks() {
+		t := tracker.Track()
+		tracker.OnEnded(func(err error) {
+			fmt.Printf("Track (ID: %s, Label: %s) ended with error: %v\n",
+				t.ID(), t.Label(), err)
```

Comment:
It works like:
```
Track (ID: video, Label: 10afd1bd-dee0-452a-85eb-b49cbea60194) ended with error: EOF
```
on camera disconnection.

BTW, ID and Label are swapped?

Reply:
Oops, I'll fix that. Here's the issue, https://github.com/pion/mediadevices/issues/32.